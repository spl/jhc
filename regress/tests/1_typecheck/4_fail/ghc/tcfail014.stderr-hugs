ERROR "tcfail014.hs":5 - Type error in application
*** Expression     : z z
*** Term           : z
*** Type           : a -> b
*** Does not match : a
*** Because        : unification would give infinite type
