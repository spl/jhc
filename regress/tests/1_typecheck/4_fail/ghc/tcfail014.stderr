
tcfail014.hs:5:33:
    Occurs check: cannot construct the infinite type: t1 = t1 -> t0
    In the first argument of `z', namely `z'
    In the expression: z z
    In an equation for `h': h z = z z
